src/components/sideNavigation/SideNavigationLoadingExample.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
```

Comment:
Place this in an `example` folder. Like other components. Otherwise, it shows up in the coverage.